.vale.ini

```diff
@@ -5,11 +5,6 @@ Vocab = Base
 
 Packages = Google
 
-# Make vale reason with unsupported file extensions in the same way as java
-[formats]
-kt = java
-kts = java
-
 [*/glossary.md]
 BasedOnStyles = Vale, Google
```

Comment:
Needs to have the tech-writing-style-guide to match the mobile-android-pipelines vale.ini

Reply:
agreed, added